Formula/couchdb.rb

```diff
@@ -120,12 +231,15 @@ def plist; <<-EOS.undent
     inreplace "#{testpath}/couchdb/default.ini", "/usr/local/var", testpath/"var"
 
     pid = fork do
+      ENV["ERL_LIBS"] = geocouch_share if build.with? "geocouch"
       exec "#{bin}/couchdb -A #{testpath}/couchdb"
     end
     sleep 2
 
     begin
-      assert_match /Homebrew/, shell_output("curl localhost:5984")
+      assert_match /Homebrew/, shell_output("curl -# localhost:5984")
+      if build.with? "geocouch"
+      end
```

Comment:
That's where I figured out I didn't know how to test `geocouch`. No, shouldn't be there.